src/components/EditorSideBar/Blueprints.tsx

```diff
@@ -1,11 +1,21 @@
 import courtList from "../ChangeCourtSize/CourtList";
 import { Image, Box } from "@chakra-ui/react";
+import { useDispatch } from "react-redux";
+1;
+import { changeCourtName } from "@/store/reducer/courtNameSlice";
+import React from "react";
+
+const Blueprints: React.FC = () => {
+  const dispatch = useDispatch();
+
+  const handleCourtName = (des: string, des1: string): void => {
+    dispatch(changeCourtName(`${des} ${des1}`));
+  };
 
-const Blueprints = () => {
   return (
     <Box pl="24px" pt="24px" height="100%" className="scrollbox">
       {courtList.map((court) => {
-        const { img } = court;
+        const { img, des, des1 } = court;
```

Comment:
It is not your problem but des and des1 is not a proper name for variables. Maybe change them?

Reply:
I do aware of this problem  I will create a new ticket to fix those naming issues at one time